org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -1,48 +1,65 @@
 define([
   'common/modules/services',
+  'common/angularFileUpload'
 ], function (module) {
   'use strict';
 
-  module.factory('FileUploadService', ['$q', 'FileUploader',
+  module.factory('FileUpload', ['$q', 'FileUploader',
```

Comment:
added `  'common/angularFileUpload'` here only and not in common.services as angularFileUpload is not relevant to other common services